frontend/src/static/js/v_summary.js

```diff
@@ -183,7 +183,7 @@ window.vSummary = {
       const params = window.location.hash.slice(1).split('&');
       params.forEach((param) => {
         const [key, val] = param.split('=');
-        window.hashParams[key] = decodeURIComponent(val);
+        if (key) { window.hashParams[key] = decodeURIComponent(val); }
```

Comment:
By our coding convention, we discourage the use of single line conditional because making changes to either (the condition or the statements) affect the other.

Please change to 

```
if (key) { 
    window.hashParams[key] = decodeURIComponent(val); 
}
```

Reply:
I'll change it. 
Since single line conditionals are currently used throughout the same method, should I change the other lines too in a seperate PR?

Reply:
> Since single line conditionals are currently used throughout the same method, should I change the other lines too in a seperate PR?

That one we can slowly fix them when there's a need to change them